components/brave_wallet_ui/options/network-options.ts

```diff
@@ -1,35 +0,0 @@
-import { NetworkOptionsType } from '../constants/types'
```

Comment:
users can add custom networks now with long names, we need to determine rules how to reduce it in the UI cc @Douglashdaniel

Reply:
Could we add a method in the `utility folder` something along these lines? 
```
export const reduceNetworkDisplayName = (name: string) => {
  const firstWord = name.split(' ')[0]
  if (firstWord.length > 9) {
    const firstEight = firstWord.slice(0, 6)
    const reduced = firstEight.concat('..')
    return reduced
  } else {
    return firstWord
  }
}
```

Reply:
done, thx